lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -373,11 +373,27 @@ class MongoClientInterface {
         }], {
             ordered: 1,
         }, err => {
+            /*
+             * Related to https://jira.mongodb.org/browse/SERVER-14322
+             * It happens when we are pushing two versions "at the same time"
+             * and the master one does not exist. In MongoDB, two threads are
+             * trying to create the same key, the master version, and one of
+             * them, the one with the highest versionID (less recent one),
+             * fails.
+             * We check here than than the MongoDB error is related to the
+             * second operation, the master version update and than the error
+             * code is the one related to mentionned issue.
+             */
             if (err) {
-                log.error(
-                    'putObjectVerCase1: error putting object version',
-                { error: err.message });
-                return cb(errors.InternalError);
+                log.debug('putObjectVerCase1: error putting object version',
```

Comment:
I would like to see this log after the line num. 395. Otherwise it will be printed twice for regular errors.

Reply:
done